geode-core/src/main/java/org/apache/geode/distributed/Locator.java

```diff
@@ -101,7 +105,7 @@
    */
   public static Locator startLocator(int port, File logFile) throws IOException {
 
-    return startLocator(port, logFile, false, (InetAddress) null, (Properties) null, true, true,
+    return startLocator(port, logFile, false, (HostAddress) null, (Properties) null, true, true,
```

Comment:
Here and in a few other places in this file (that call startLocator) casting null args to HostAddress or Properties seems redundant. Is there a reason to keep that anyway?